vendor/golang.org/x/sys/unix/syscall_linux.go

```diff
@@ -73,6 +77,12 @@ func IoctlGetInt(fd int, req uint) (int, error) {
 	return value, err
 }
 
+func IoctlGetWinsize(fd int, req uint) (*Winsize, error) {
```

Comment:
__[golint]__
* [golint] exported function IoctlGetWinsize should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->